CMakeLists.txt

```diff
@@ -441,6 +459,10 @@ install(FILES
   config/sensors_dummy.yaml
   config/surface_segmentation.yaml
   launch/editor.launch
+  launch/baxter_app.launch
+  launch/baxter.launch
+  ## launch/baxter_moveit.launch
+  launch/baxter_planning_context.launch
```

Comment:
Remove this (see note on the file itself).

Reply:
You mean remove planning_context.launch?